buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -41,29 +42,29 @@ object Versions {
 
   const val com_github_triplet_play_gradle_plugin: String = "2.5.0" // available: "2.6.1"
 
-  const val kotlin_annotation_processing_gradle: String = "1.3.60"
+  const val kotlin_annotation_processing_gradle: String = "1.3.61"
 
-  const val kotlin_android_extensions_runtime: String = "1.3.60"
+  const val kotlin_android_extensions_runtime: String = "1.3.61"
 
-  const val kotlin_android_extensions: String = "1.3.60"
+  const val kotlin_android_extensions: String = "1.3.61"
 
   const val multidex_instrumentation: String = "2.0.0"
 
   const val javax_annotation_api: String = "1.3.2"
 
-  const val kotlin_gradle_plugin: String = "1.3.60"
+  const val kotlin_gradle_plugin: String = "1.3.61"
 
   const val logging_interceptor: String = "3.12.1" // available: "4.2.2"
 
   const val ink_page_indicator: String = "1.3.0"
 
-  const val kotlin_stdlib_jdk7: String = "1.3.60"
+  const val kotlin_stdlib_jdk7: String = "1.3.61" // available: "1.3.61"
 
-  const val leakcanary_android: String = "2.0-alpha-2"
+  const val leakcanary_android: String = "2.0-alpha-2" // available: "2.0"
```

Comment:
We can upgrade these to stable versions.

Reply:
done